migrations/0005_create_orders.up.sql

```diff
@@ -0,0 +1,21 @@
+create table orders (
+  id uuid primary key not null default uuid_generate_v4(),
+  created_at timestamp with time zone not null default current_timestamp,
+  updated_at timestamp with time zone not null default current_timestamp,
+  total_price numeric(28, 18) not null,
+  merchant_id text NOT NULL,
+  status text NOT NULL CHECK (status IN ('pending', 'paid', 'fulfilled', 'canceled'))
```

Comment:
I would make this check separate so it is easier to change later

Reply:
https://github.com/brave-intl/bat-go/pull/226/commits/f0373b4d3330cfc0ff6dcae800a758bd533d808c